### DISCUSSION TITLE
Linking directly to markdown blocks

### DISCUSSION THREAD

**Discussion by AlexandreNarten** [docmost/docmost#180] [Ideas]
I've been using [ClickUp](https://clickup.com/) for a while now and have found that the ability to link directly to a text block or header or image very useful. Is this a feature that has been considered before ?

Kind regards,
Alexandre.

**Comment by quistuipater** [comment]
I second the need for this feature: Named anchors are essential to any large project or document.

**Comment by SyndicatedPillbug** [comment]
I'd love this as well!

**Reply by helderrscorreia** [reply]
Me too +1

**Comment by drago1520** [comment]
You can do it. Make a heading -> hover over the heading, on the right there's a link icon -> click it.
<img width="355" height="195" alt="image" src="https://github.com/user-attachments/assets/34197e6e-96b8-47aa-b296-9f7fdcd4f55a" />

It must be a Heading & u must hover over it for the icon to appear
